packages/story-editor/src/app/rightClickMenu/provider.js

```diff
@@ -246,88 +257,148 @@ function RightClickMenuProvider({ children }) {
 
   /**
    * Send element one layer backwards, if possible.
+   *
+   * @param {Object} evt The triggering event
    */
-  const handleSendBackward = useCallback(() => {
-    const newPosition =
-      currentPosition === 1 ? currentPosition : currentPosition - 1;
+  const handleSendBackward = useCallback(
+    (evt) => {
+      const newPosition =
+        currentPosition === 1 ? currentPosition : currentPosition - 1;
 
-    arrangeElement({
-      elementId: selectedElement.id,
-      position: newPosition,
-    });
-  }, [arrangeElement, currentPosition, selectedElement?.id]);
+      arrangeElement({
+        elementId: selectedElement.id,
+        position: newPosition,
+      });
+
+      trackClick(evt, `context_menu_${selectedElementType}_send_backward`);
```

Comment:
🤔 is it helpful to have the element's type added here? If not I can remove this.

Reply:
It is. But instead of adding it to the event name, this should be a custom param.

Second, since we're not tracking clicks on external links, we should use `trackEvent` instead with custom params.

So this would become something like:


```js
trackEvent('context_menu_action', {
  name: 'send_backward',
  element: selectedElementType
});
```

So no need for passing `evt` everywhere

Reply:
Can do!